File: AngularApp/src/app/swagger-proxy/models/ledger-transaction-result-dto.ts
/* tslint:disable */
import { LedgerTransactionResultTypeEnum } from './ledger-transaction-result-type-enum';
import { LedgerTransactionDto } from './ledger-transaction-dto';
export interface LedgerTransactionResultDto {
  resultType: LedgerTransactionResultTypeEnum;
  transactionData?: LedgerTransactionDto;
}

File: Domain/LedgerTransactions/LedgerTransactionResultDto.cs
namespace Domain.LedgerTransactions
{
    public class LedgerTransactionResultDto
    {
        public LedgerTransactionResultTypeEnum ResultType { get; set; }
        public LedgerTransactionDto TransactionData { get; set; }

        public decimal AccountBalance { get; set; }
    }
}
